src/main/java/jenkins/security/plugins/ldap/LDAPConfiguration.java

```diff
@@ -472,6 +495,55 @@ private static String addPrefix(String server) {
         else return "ldap://" + server;
     }
 
+    private String generateId() {
+        return generateId(server, rootDN, userSearchBase, userSearch);
+    }
+
+    @Restricted(NoExternalUse.class)
+    static String generateId(String serverUrl, String rootDN, String userSearchBase, String userSearch) {
+        final MessageDigest digest = DigestUtils.getMd5Digest();
```

Comment:
Is it worth all this instead of `Util.getDigestOf(normalizeServer(serverUrl)+...+...)`